app.rb

```diff
@@ -0,0 +1,91 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'sinatra/flash'
```

Comment:
I cloned and did bundle install but when I run rackup I am getting an error that says 'cannot load such file -- sinatra/flash (LoadError)'. I think sinatra-flash may be missing from the gem file